src/monaco-treemending.ts

```diff
@@ -67,6 +77,11 @@ async function run () {
     })
   })
 
+  // in case of pnpm copy the temp esm back
+  if (monacoEsmDirectory.includes('.pnpm')) {
```

Comment:
nit: do we need to check if the file exist twice?

Reply:
No, you are right. Give me a minute